tests/conftest.py

```diff
@@ -26,6 +26,15 @@
 from invenio_vocabularies.records.models import VocabularyScheme
 
 
+@pytest.fixture(scope='module')
+def app_config(app_config):
```

Comment:
Not ideal. But I did not find any other way to be able to get/pass as header the CSRF token. Otherwise, any POST/PUT/PATCH/DELETE call will fail.